packages/terra-form-input/tests/jest/Input.test.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Input from '../../src/Input';
+
+describe('Input', () => {
+  const defaultRender = <Input />;
```

Comment:
Can we add snapshots for `required` as well.

Reply:
Done 159643ece2f150a8c2f2e00196c9f92a88254c2d